<launch>
    <node pkg="au_vision" type="detector_server" name="ArcDetector" output="screen">
        <param name="detector" value="arc_detector"/>
        <!--remap from="TestTracker/camera" to="/usb_cam/image_raw"/-->
    </node>
</launch>
